sds_data_manager/stacks/sds_data_manager_stack.py

```diff
@@ -54,25 +52,24 @@ def __init__(
         ----------
         scope : App
         construct_id : str
-        sds_id : str
-            Name suffix for stack
         opensearch: OpenSearch
             This class depends on opensearch, which is built with opensearch_stack.py
         dynamodb_stack: DynamoDb
             This class depends on dynamodb_stack, which is built with
             opensearch_stack.py
         processing_step_function_arn:
             This has step function arn
-        env : Environment
-            Account and region
         """
         super().__init__(scope, construct_id, env=env, **kwargs)
+        # Get the current account number so we can use it in the bucket names
+        account = env.account
+        region = env.region
 
         # This is the S3 bucket used by upload_api_lambda
         self.data_bucket = s3.Bucket(
             self,
-            f"DataBucket-{sds_id}",
-            bucket_name=f"sds-data-{sds_id}",
+            "DataBucket",
+            bucket_name=f"sds-data-{account}",
```

Comment:
Right now this bucket_name is sds-data-<account number>. Change it to <account_name> instead.

Reply:
I don't think `sds-data-dev` is unique enough, and may be taken by someone else. I chose the account number because it is sort of a unique-ish value we can count on being available within each account (if you want to deploy this stack to your own "dev" account for instance). We don't care what the bucket is named really as long as we can access it programmatically.

Reply:
Alright that's fine.